spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -164,19 +165,26 @@ public TaskExecutionCreationService taskExecutionRepositoryService(TaskRepositor
 		return new DefaultTaskExecutionRepositoryService(taskRepository);
 	}
 
+	@Bean
+	TaskAppDeploymentRequestCreator taskAppDeploymentRequestCreator(
```

Comment:
Is there any reason why this is not `public`?

Reply:
ooops update!